ayan-biswas0412.md

```diff
@@ -0,0 +1,49 @@
+---
+name: "@ayan-biswas0412"
+project: "Camera Based Door Access Control System"
+---
+
+# Camera Based Door Access Control System
+
+## Summary
+
+I am going to build a camera based door access control system that uses facial recognition to grant or deny access to a locked door. 
+The system will include a camera mounted near the door, a small computer for processing the video feed and running the facial recognition algorithm, and a lock mechanism that can be controlled by the computer. 
+I am excited to build this because I think it has the potential to improve security and convenience in a variety of settings, such as office buildings, apartment complexes, and homes.
+
+## Plan
+
+1. Gather materials and tools needed:
+    * Raspberry Pi microcomputer
+    * Raspberry Pi camera module
+    * Servo motor
+    * Lock mechanism (such as a deadbolt or latch bolt)
+    * Wires and connectors
+    * Power supply
+2. Set up the Raspberry Pi and camera module according to the manufacturer's instructions.
+3. Connect the servo motor to the Raspberry Pi and wire it to control the lock mechanism.
+4. Write and upload code to the Raspberry Pi that will:
+    * Continuously capture video from the camera
+    * Run the facial recognition algorithm on the video feed
+    * Grant or deny access to the door based on the result of the facial recognition
+5.Test the system to ensure it is functioning properly.
+6. Install the camera, servo motor, and lock mechanism near the desired door.
+
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Raspberry Pi 3 - Model B - ARMv8 with 1G RAM   | [https://www.adafruit.com/product/3055](https://www.adafruit.com/product/3055) | $35  |
```

Comment:
it looks like this is out of stock, do you have another source/site where you'd be able to purchase the pi?

Reply:
also, do you need an SD card?

Reply:
No I don't need an SD card, and I have Amazon as my alternative from where I will be purchasing if this goes out of stock in adafruit.

Reply:
sounds good, thanks!